banco.sql

```diff
@@ -538,8 +538,8 @@ CREATE TABLE `anotacoes_os` (
 CREATE TABLE `configuracoes` ( `idConfig` INT NOT NULL AUTO_INCREMENT , `config` VARCHAR(20) NOT NULL UNIQUE, `valor` VARCHAR(20) NOT NULL , PRIMARY KEY (`idConfig`)) ENGINE = InnoDB;
 
 INSERT INTO `configuracoes` (`idConfig`, `config`, `valor`) VALUES
-(2, 'app_name', 'Map-OS'),
-(3, 'app_theme', 'white'),
+(2, 'app_name', 'MAPOS'),
+(3, 'app_theme', 'black'),
```

Comment:
Na minha opinião o ideal é deixar como padrão o tema branco.

Reply:
_Discordo, pois se tratar da questão de saúde, o tema branco implica em maior desgaste visual por longo período de tempo de uso. a menos que diminuam a tonalidade da cor aplicada._

Reply:
Não exatamente, pois segundo pesquisas o contrário que é o verdadeiro. Vários estudos datando dese 1980 chegam a resultados que fundo claro com letras escuras é o melhor para o olho humano: https://ux.stackexchange.com/questions/53264/dark-or-white-color-theme-is-better-for-the-eyes